### PR TITLE
Add variable in shell element: Shear angle

### PR DESCRIPTION
**Pull request by PhilippeBussetta**
I have had a variable into the thin shell element and the possibility to use a null Young modulus with the CL elastic_isotropic_plane_stress_uncoupled_shear

### CODE REVIEW

File: applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_plane_stress_uncoupled_shear.cpp [hunk 1]
```diff
@@ -62,7 +62,23 @@ int ElasticIsotropicPlaneStressUncoupledShear::Check(
     const ProcessInfo& rCurrentProcessInfo
 )
 {
-    LinearPlaneStress::Check(rMaterialProperties, rElementGeometry, rCurrentProcessInfo);
+    if (YOUNG_MODULUS.Key() == 0 || rMaterialProperties[YOUNG_MODULUS] < 0.0)
+    {
+        KRATOS_ERROR << "YOUNG_MODULUS has Key zero or invalid value " << std::endl;
```
Comment (loumalouomega) [inline applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_plane_stress_uncoupled_shear.cpp]: Use the checks in includes/checks.h

~~~cpp
KRATOS_CHECK_VARIABLE_KEY(YOUNG_MODULUS)
~~~
and KRATOS_ERROR_IF

**Review by loumalouomega — CHANGES_REQUESTED** [review]
Minor change

File: applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_plane_stress_uncoupled_shear.cpp [hunk 2]
```diff
@@ -62,7 +62,23 @@ int ElasticIsotropicPlaneStressUncoupledShear::Check(
...
+    }
+
+    const double& nu = rMaterialProperties[POISSON_RATIO];
+    const bool check = bool((nu >0.499 && nu<0.501) || (nu < -0.999 && nu > -1.01));
```
Comment (philbucher) [inline applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_plane_stress_uncoupled_shear.cpp]: static cast

**Review by philbucher — COMMENTED** [review]
General comment:
I am continuously cleaning the shells and unifying/moving things to the BaseClass (`BaseShellElement`)

This feature is only added to one shell and there is no test for it, so I cannot promise for it to survive.

If at some point it stops working then it is likely that I moved something to the baseclass, in this case you would have to reimplement it in the baseclass

**Comment by PhilippeBussetta** [comment]
done

**Review by loumalouomega — DISMISSED** [review]
Thanks, we need to clean the rest of the CL :P consistently

**Comment by PhilippeBussetta** [comment]
@philbucher  It is not a problem, it is a small computation easy to reimplement.
I will leave my job soon and I would like to merge this before.
Thanks for your help

**Review by philbucher — DISMISSED** [review]
ok :+1:

**Comment by PhilippeBussetta** [comment]
@loumalouomega 
The worst implementation is always copied/pasted ;-)

**Comment by loumalouomega** [comment]
We can call that Busseta's Law, like Murphy's Law

**Review by loumalouomega — DISMISSED** [review]

**Comment by loumalouomega** [comment]
I removed an extra ;

**Comment by PhilippeBussetta** [comment]
I removed other extra ;

**Review by loumalouomega — APPROVED** [review]

**Pull request merged**